README.md

```
@@ -308,7 +309,10 @@
 - *"Change upstream app for app.subdomain.conf to newapp"*
 - *"Update app.subdomain.conf to use newapp:8081"*
 - *"Modify plex config to use port 32401"*
+- *"Add MCP endpoint to jellyfin.subdomain.conf"*
+- *"Add MCP location at /ai-service to plex.subdomain.conf"*
 
+#### `list` Action
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)